doc/source/release/1.16.0-notes.rst

```diff
@@ -127,8 +153,18 @@ Array API Standard Support
 Experimental support for array libraries other than NumPy has been added to
 existing sub-packages in recent versions of SciPy. Please consider testing
 these features by setting an environment variable ``SCIPY_ARRAY_API=1`` and
-providing PyTorch, JAX, or CuPy arrays as array arguments.
-Features with support added for SciPy 1.16.0 include:
+providing PyTorch, JAX, or CuPy arrays as array arguments. Many functions
+in `scipy.stats`, `scipy.special`, `scipy.optimize`, and `scipy.constants`
+now provide tables documenting compatible array and device types as well as
+support for lazy arrays and JIT compilation. New features with support and
+old features with support added for SciPy 1.16.0 include:
+
+- Most features of `scipy.signal`
+- `scipy.ndimage.vectorized_filter`
+- `scipy.special.stdtrit`
```

Comment:
```suggestion
- `scipy.special.logsumexp`
- `scipy.special.softmax`
- `scipy.special.stdtrit`
```

Reply:
Also added before this relase, right?

Reply:
Neither worked in jax.jit or Dask.

Reply:
For the record: as much as end users are concerned, "worked in jax.jit" and "worked with JAX" should be treated almost as synonims. Not being able to apply the jit makes JAX very undesirable compared to numpy or cupy.

Reply:
Same as https://github.com/tylerjereddy/scipy/pull/83#discussion_r2093759143.